modules/oaiharvest/lib/oai_harvest_daemon.py

```diff
@@ -572,36 +572,47 @@ def remove_duplicates(harvested_file_list):
     """
     Go through a list of harvested files and remove any duplicate records.
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
WRT throwing a warning: I'll have a look at this. I understand that I should just fail here (gracefully ;) if lxml is not installed.

WRT 1.2 I had a separate pull request for master as the code changed there a bit.